core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -148,6 +173,79 @@ oppia.directive('tutorCard', [
             });
           };
 
+          $scope.playPauseAudioTranslation = function() {
+            // TODO(tjiang11): Change from on-demand loading to pre-loading.
+
+            if (cardIndexOfPlayingAudio === undefined) {
+              cardIndexOfPlayingAudio =
+                playerPositionService.getActiveCardIndex();
+
+              // TODO(tjiang11): On first play, ask learner to pick language
+              // and subsequently for confirmation to use bandwidth 
+              // to download audio files.
+            }
+
+            $scope.showExtraAudioControls = true;
+
+            if (!AudioPlayerService.isPlaying()) {
+              if (AudioPlayerService.trackLoaded() && isSameAudio()) {
+                AudioPlayerService.play();
+              } else {
+                loadAndPlayAudioTranslation();
+              }
+            } else {
+              AudioPlayerService.pause();
+              if (!isSameAudio()) {
+                loadAndPlayAudioTranslation();
```

Comment:
So without this line this is the behavior: You're playing audio on a card, while the audio is still playing, you switch to another card, and you click the audio button on another card (since it's another card it's a play button and not a pause button). Then the audio for the other card stops. You have to click a second time to start the audio for the new card. 

With this line the audio from the old card stops and this audio starts playing immediately. Do you think that's okay?